src/connector/src/parser/mod.rs

```diff
@@ -46,6 +46,7 @@ pub use self::mysql::mysql_row_to_owned_row;
 use self::plain_parser::PlainParser;
 pub use self::postgres::postgres_row_to_owned_row;
 use self::simd_json_parser::DebeziumJsonAccessBuilder;
+pub use self::sql_server::sql_server_row_to_owned_row;
```

Comment:
not used in this file?

Reply:
yes. Just to expose `sql_server_row_to_owned_row` as `parser::sql_server_row_to_owned_row` instead of `use parser::sql_server::sql_server_row_to_owned_row` like the `postgres_row_to_owned_row` and `mysql_row_to_owned_row`